git_machete/docs.py

```diff
@@ -5,6 +5,7 @@
     "add": "Add a branch to the tree of branch dependencies",
     "advance": "Fast-forward merge one of children to the current branch and then slide out this child",
     "anno": "Manage custom annotations",
+    "clean": "Delete untracked and unmanaged branches with no downstream branch and also check out user's open GitHub PRs",
```

Comment:
For the sake of top-level summary, maybe skip this detail:
```suggestion
    "clean": "Delete untracked and unmanaged branches and also check out user's open GitHub PRs",
```
(also it's now somewhat confusing, coz `with no downstream branch` applies to `unmanaged`, while it should apply to `untracked`)